docs/bulletin.md

```diff
@@ -1,6 +1,9 @@
 # Bulletin
 
 ## SLEAP v1.3.2
+
+_mm/dd/yyyy_
+
```

Comment:
I would have added this to the first PR on [shrivaths/changelog-announcement-1](https://github.com/talmolab/sleap/tree/shrivaths/changelog-announcement-1), then do a `git pull origin shrivaths/changelog-announcement-1` to update this branch. Just to keep changes organized.
```suggestion

```